### PR TITLE
New role Taskinator and made mario incompatible with unlucky

### PR DESCRIPTION
**Pull request by ryuk201198**
Review please

### CODE REVIEW

File: Roles/Neutral/Taskinator.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        taskIndex = new();
+        TaskMarkPerRound = new();
+        IsEnable = false;
+        maxTasksMarkedPerRound = TaskMarkPerRoundOpt.GetInt();
+    }
+    public static void Add(byte playerId)
+    {
+        playerIdList.Add(playerId);
+        TaskMarkPerRound[playerId] = 0;
+        IsEnable = true;
```
Comment (Tommy-XL) [inline Roles/Neutral/Taskinator.cs:40]: Seems `playerIdList` it's not used anywhere, so it can be removed

File: main.cs [hunk 2]
```diff
@@ -1008,6 +1010,7 @@ public enum AdditionalWinners
     Phantom = CustomRoles.Phantom,
     Maverick = CustomRoles.Maverick,
     Shaman = CustomRoles.Shaman,
+    Taskinator = CustomRoles.Taskinator,
```
Comment (Tommy-XL) [inline main.cs:1013]: You forgot add `AdditionalWinners.Taskinator` in `en_US.json`

File: Roles/Neutral/Taskinator.cs (hunk 1 above)
Comment (ryuk201198) [reply Roles/Neutral/Taskinator.cs:40]: I plan on using it in the future, for a different role

Comment (Tommy-XL) [reply Roles/Neutral/Taskinator.cs:40]: Hmm, ok

File: main.cs (hunk 2 above)
Comment (ryuk201198) [reply main.cs:1013]: done

**Review by Tommy-XL — APPROVED** [review]

**Pull request merged**